backend/src/cmd/import_realm_tree.rs

```diff
@@ -40,6 +40,12 @@ struct Realm {
 
     #[serde(default)]
     children: Vec<Realm>,
+
+    #[serde(default)]
+    page_admins: Option<Vec<String>>,
+
+    #[serde(default)]
+    page_moderators: Option<Vec<String>>,
```

Comment:
The `#[serde(default)]` is likely not necessary if it's already an `Option`. So either could be removed. Looking at the code below, I think it's better to remove the `Option`, i.e. make the type `Vec<String>`.